code/modules/research/designs/autolathe_desings/autolathe_designs_construction.dm

```diff
@@ -106,4 +106,5 @@
 	build_type = AUTOLATHE
 	materials = list(MAT_METAL = 12000, MAT_GLASS=8000)
```

Comment:
hey make sure you tell it its a prolathen thing and not just autolathen.
Also this can lead to explotes do to t1 being 3 metal 2 glass and t4 being less then 1 sheet of each